app/scripts/directives/podMetrics.js

```diff
@@ -358,6 +358,11 @@ angular.module('openshiftConsole')
               // get the unit value if specified
               var unit =  metric.unit || "";
 
+              // A typical metric ID is of the form "pod/<pod-id>/custom/<some-endpoint-info>"
+              // such as "pod/be381d4b-87fc-11e5-b2a3-525400b33d1d/custom/JVM-Heap-Memory-Used".
```

Comment:
> @mwringe Do you know if the format we assume in that function always holds?

It should hold. We control what the metric id looks like coming from the agent and from heapster.

Technically metric id should be considered more opaque, but we understand that some clients are expecting a certain format and constructing the metric id directly instead of using tag queries.